### ISSUE TITLE
Ability for Screen Readers to Navigate by page

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#25484]
### Your idea

Currently there is no way for a screen reader user to navigate to the next/previous page.  Page Up/Down keys and Ctrl + Page up/Down only scroll the view but do not move the focus.

Suggest that Ctrl + Page Up/Down moves the focus to the first note of the next/previous page.  Leaving just Page up/Down to scroll the view for sighted users.

Alternatively a second option could be to assign Alt + Page Up/Down to move to next/previous page, shifting focus to the first note and also allow this to be a customisable shortcut so a blind user can change it to just Pg Up/Down if they desire as we have no need to be able to scroll the view without moving focus.

### Problem to be solved

Allow blind users to be able to navigate by page.  Currently there is no way of knowing where page breaks are or moving to them.  See also #23780 

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by MarcSabatella** [comment]
Yes.  Both PgDn and Ctrl+PgDn scroll the view only without changing the selection.  This is great for sighted users wanting to, for instance, select a note in the score and then scroll the view further down in order to Shift+click another note to create a range selection.  But this not useful for blind users, and having a way to actually navigate to the next page - change the selection - is more important.

My own sense is that there is no particular reason Ctrl+PgDn couldn't be modified to select the first element of the new page, while leaving the scroll-only behavior of PgDn alone.  This would also mimic the behavior of Home & End, where they reposition the view only but with Ctrl they select as well.  I kind of doubt people actually use Ctrl+PgDn often for creating range selections.  But FWIW, also no reason we couldn't support Shift to extend the selection - so Ctrl+Shift+PgDn to extend the selection to the next page, just as Ctrl+Shift+End extends to end of score.
